packages/wrangler/src/deployment-bundle/find-additional-modules.ts

```diff
@@ -49,16 +50,41 @@ export async function findAdditionalModules(
 			name: m.name,
 		}));
 
+	// Try to find a requirements.txt file
+	const isPythonEntrypoint =
+		getBundleType(entry.format, entry.file) === "python";
+
+	if (isPythonEntrypoint) {
+		try {
+			const pythonRequirements = await readFile(
+				path.resolve(entry.directory, "requirements.txt"),
+				"utf-8"
+			);
+
+			// This is incredibly naive. However, it supports common syntax for requirements.txt
+			for (const requirement of pythonRequirements.split("\n")) {
+				modules.push({
+					type: "python-requirement",
+					name: path.resolve(entry.moduleRoot, requirement),
+					content: "",
+					filePath: undefined,
+				});
+			}
+			// We don't care if a requirements.txt isn't found
+		} catch (e) {}
```

Comment:
Should we log the error as a warning? or alternatively be more explicit about which exceptions we are ignoring?